lib/rust/ensogl/core/src/display/scene.rs

```diff
@@ -937,6 +949,8 @@ impl SceneData {
         // setups now, so we are using here the main camera only.
         let camera = self.camera();
         let fullscreen_vis_camera = self.layers.panel.camera();
+        // We are using the same camera as `fullscreen_vis`.
+        let welcome_screen_camera = self.layers.panel.camera();
```

Comment:
I think this comment does not hit the nail on the head. The fact that we use the fullscreen_vis_camera is rather contingent; the point is we want to have an unnavigable camera.

Reply:
I second that. This sounds like we want welcome screen to move together with fullscreen camera, which is never the case as the camera should never move.

Reply:
Reworded it, thank you